src/main/java/controllers/result/dto/KvisResultAPIDTO.java

```diff
@@ -0,0 +1,126 @@
+package controllers.result.dto;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import java.sql.Timestamp;
+import java.util.StringJoiner;
+
+/**
+ * @author Alfred Röttger Rydahl
+ * @date 26/11/2021
+ **/
+@JsonDeserialize(builder = KvisResultAPIDTO.Builder.class)
+public class KvisResultAPIDTO
+{
+	public final String id;
+	public final String kvisId;
+	public final String name;
+	public final Timestamp kvisStarted;
```

Comment:
Hvorfor skal vi have dette? Så skal jeg til at holde styr på hvornår en kvis er startet og sluttet? Kan det ikke være lige meget?

Reply:
Ja, det var noget vi diskuterede for 2 uger siden, så vidt jeg husker. Det var der hvor vi virkelig havde tavlen igang, og vi blev enige om, at vi godt ville vide hvor længe de har været om at svarer på en Kvis :smile:

Reply:
Okay, jeg synes bare at det er mærkeligt, fordi jeg så skal holde styr på det i frontend, og synes personligt det er lidt irrelevant, men hvis i gerne vil have det, så må det jo laves

Reply:
Jamen altså, hvis det er så svært at tilføje i frontend, så må vi jo lige spørge de andre om vi ikke skal fjerne det?

Reply:
Det er ikke svært, men det tager bare længere tid

Reply:
Okay, så beholder vi det og jeg merger med det samme :)